### DISCUSSION TITLE
post/be-your-authentic-self

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#22] [General]
# post/be-your-authentic-self

是的，我还活着，哈哈哈 实话实话，明天腾讯 lastday 了 之前欠了不少博客没写，实习这 6 个月也精力了不少事情 但目前我要花一段时间搞定我的英语，因为我打算出国留学，但是英语还没考到我想要的分数 所以，等我过段时间吧～我打算做回原汁原味的自己了

https://nickxu.me/post/be-your-authentic-self

**Comment by Kennems** [comment]
为什么不直接进大厂呢，是打算美硕 + 在美找工作吗？

**Reply by nickxudotme** [reply]
人生主打一个体验，这六个月我该体验的也都体验到了，已经对大厂祛魅了
我有个同学去腾讯实习了半年，又去字节实习了半年，现在已经不想上班了，大四辞职宅在学校享受校园生活，也没有课，天天在宿舍打瓦
我非常讨厌别人剧透我要玩的游戏，还好有一个游戏是最少被剧透的，也就是你的 real life
然而一眼望的到头的生活无异于品尝已经剧透的游戏，如果你要我留在国内过这种生活，我是不愿意的
当然你可以说等你老了你就知道稳定的重要了，进体制或者公务员/国企，我只能说这样的话更要趁年轻的时候多体验一下东西
我在中国活了 21 年，我为什么不能也去美国体验活个 21 年，我为什么不能去日本，去台湾，去德国，或者去澳大利亚？
简而言是，人生主打一个体验，反正 2w 多天就要死了，我希望我的未来永远是未知而充满想象力的，这样才有意思
